core/tests/protractor_desktop/ExplorationTranslationTab.js

```diff
@@ -22,40 +22,27 @@ var general = require('../protractor_utils/general.js');
 var users = require('../protractor_utils/users.js');
 var workflow = require('../protractor_utils/workflow.js');
 
-
-var ExplorationEditorPage =
-  require('../protractor_utils/ExplorationEditorPage.js');
+var CreatorDashboardPage = require('../protractor_utils/CreatorDashboardPage');
```

Comment:
Filename is missing `.js` extension. Did you run the tests locally before pushing the changes?

Reply:
This won't fail the test as the require function will append `.js` in the end, still, I think we should have the complete file name. Thanks for poiniting this out @apb7. 

![image](https://user-images.githubusercontent.com/16653571/55193953-5c96a480-51ce-11e9-9653-7f6eb5eddb92.png)

Reply:
done